src/client/assets/styles/editor.scss

```diff
@@ -13,6 +14,36 @@
         }
     }
 
+    view-mode {
+        .excel {
+            @include theme($excel);
+        }
+
+        .word {
+            @include theme($word);
+        }
+
+        .powerpoint {
+            @include theme($powerpoint);
+        }
+
+        .onenote {
+            @include theme($onenote);
+        }
+
+        .project {
+            @include theme($project);
+        }
+
+        .teams {
+            @include theme($teams);
+        }
+
+        .web {
+            @include theme($web);
+        }
```

Comment:
As of yesterday, we now need to include "Outlook" too.

Is it possible to perhaps have this bit of code elsewhere or re-use an existing bit of similar logic, so we don't need to remember to do it every time?  If not, it's OK, but in that case we should at least have a comment in both this block and the other, saying that "be sure to change the other location as well when you add a new theme".

Or maybe we can pull out both bits of code into their own file, a "global-host-to-theme-mapping.scss" or whatever...